scripts/missions/sandoria/6_2_Ranperres_Final_Rest.lua

```diff
@@ -118,13 +125,8 @@ mission.sections =
 
                     if
                         missionStatus == 1 and
-                        not GetMobByID(krtID.mob.CORRUPTED_YORGOS):isSpawned() and
-                        not GetMobByID(krtID.mob.CORRUPTED_SOFFEIL):isSpawned() and
-                        not GetMobByID(krtID.mob.CORRUPTED_ULBRIG):isSpawned()
+                        npcUtil.popFromQM(player, npc, ranparesRestTable, { claim = true, hide = 0, })
```

Comment:
Does this have an impact if sneak pulling? Which is possible here

Reply:
Good point - removing the claim = true should negate that.